src/topp/DecoyDatabase.cpp

```diff
@@ -73,7 +85,7 @@ class TOPPDecoyDatabase :
   {
     registerInputFileList_("in", "<file(s)>", ListUtils::create<String>(""), "Input FASTA file(s), each containing a database. It is recommended to include a contaminant database as well.");
     setValidFormats_("in", ListUtils::create<String>("fasta"));
-    registerOutputFile_("out", "<file>", "", "Output FASTA file where the decoy database will be written to.");
+    registerOutputFile_("out", "<file>", "", "Output FASTA file where the decoy database or neighbor peptide will be written to.");
```

Comment:
Its not clear when neighbor peptides are generated.
Add this info here (short version) and in the general documentation section above.